src/routes/intro/ProfessionSelectionScreen.tsx

```diff
@@ -48,9 +44,10 @@ const ProfessionSelectionScreen = ({ route, navigation }: ProfessionSelectionScr
   const { discipline } = route.params
   const { data: disciplines, error, loading, refresh } = useLoadDisciplines(discipline)
   const { data: selectedProfessions, refresh: refreshSelectedProfessions } = useReadSelectedProfessions()
+  const isProfessionSelected = selectedProfessions && selectedProfessions.length > 0
```

Comment:
Optional: Imo naming should be plural, otherwise I'd think it is just whether a currently displayed profession is selected and not whether any profession is selected.